docs/_09_development/_develop_guide/_develop_guide/_setting_code.rst

```diff
@@ -0,0 +1,100 @@
+.. _development_guide_code:
+
+-----------------------------
+Setting Up Visual Studio Code
+-----------------------------
+
+A set of tools can be configured on Visual Studio Code to help adjusting your code to the Aerostack2 Code Style guidelines during the development proccess.
+
+.. _development_guide_code_extensions:
+
+Install required Extensions
+===========================
+
+Install the following required extensions from the 'Extensions' tab in VSCode:
+
+* ``zachflower.uncrustify``
+* ``mine.cpplint``
+* ``ms-python.autopep8``
+* ``ms-python.pylint``
+
+
+.. _development_guide_code_settings:
+
+Modify 'settings.json'
+======================
+
+Open your ``settings.json`` file from VSCode and add the following content to it:
+
+.. code-block:: json
+
+    // cpp
+    "[cpp]": {
+        "editor.defaultFormatter": "zachflower.uncrustify",
+        "editor.codeActionsOnSave": {
+            "source.fixAll": "explicit"
+        },
+        "editor.formatOnSave": true
+    },
+    "uncrustify.configPath.linux": "<path-to>/.uncrustify.cfg",
+
+    // cpplint
+    "cpplint.cpplintPath": "<path-to-bin>/cpplint",
+    "cpplint.lineLength": 100,
+    "cpplint.verbose": 0,
+    "cpplint.filters": [
+        // header guards are user defined
+        "-build/header_guard",
+        // we do allow C++11
+        "-build/c++11",
+        // we consider passing non-const references to be ok
+        "-runtime/references",
+        // we wrap open curly braces for namespaces, classes and functions
+        "-whitespace/braces",
+        // we don't indent keywords like public, protected and private with one space
+        "-whitespace/indent",
+        // we allow closing parenthesis to be on the next line
+        "-whitespace/parens",
+        // we allow the developer to decide about whitespace after a semicolon
+        "-whitespace/semicolon",
+    ],
+
+    // python
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.autopep8",
+        "editor.formatOnSave": true,
+    },
+    "autopep8.args": ["--max-line-length=99", "--ignore=''"],
+
+    // ROS2 uses as a linter flake8==4.0.1 (apt installation). Pip installation should be 
```

Comment:
Afaik, this issue happens only with ROS 2 Humble. Newer versions of ROS 2 include upgraded flake8 (in ament_flake8). Thus, flake8 might be used from code too with the following settings:
```
    // flake8
    "flake8.args": ["--extend-ignore=B902,C816,D100,D101,D102,D103,D104,D105,D106,D107,D203,D212,D404,I202",
                    "--import-order-style=google", "--max-line-length=99", "--show-source=true",
                    "--statistics=true"],
```